### ISSUE TITLE
csv files for custom rain fails with line ending CRLF fails with unrelated ERROR message

### ISSUE THREAD

**Issue by Jonasvsl** [nens/threedi-api-qgis-client#85]
* threedi-api-qgis-client version:
* Python version:
* Operating System:

### Description
When trying to use a custom rain event in the plugin it fails if the file has a windows EOL of CRLF
In unix format it works.
The message is a bit unclear for users: 

![image](https://user-images.githubusercontent.com/1453159/86033033-9b05b980-ba38-11ea-841c-d724445d16f4.png)

but also we should allow users to be able to use CRLF endings

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

**Comment by Jonasvsl** [comment]
[YS2017.zip](https://github.com/nens/threedi-api-qgis-client/files/4850766/YS2017.zip)

**Comment by DamnBack** [comment]
Fixed in the version 2.0.0
